source/Cosmos.Core/Memory/HeapSmall.cs

```diff
@@ -449,9 +471,10 @@ public static void Free(void* aPtr)
             if (size == 0)
             {
                 // double free, this object has already been freed
-                Debugger.DoBochsBreak();
-                Debugger.DoSendNumber((uint)heapObject);
-                Debugger.SendKernelPanic(0x99);
+                //Debugger.DoBochsBreak();

```

Comment:
Can you please undo all the commenting out of the code?

Reply:
Why is need to crash the OS If you accidentaly freeing something not allocated?

Reply:
My use case is that if we fail hard it's easier to figure out what's going wrong. This prevents us having difficult to find bugs in the garbage collector for example. For a more user-friendly of the function, i would suggest checking if it's not already freed.

Reply:
I changed it but in way that the user can ignore the kernel panic is is not fail